hr_holidays_imposed_days/models/hr_imposed_holidays.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class HrHolidaysImposed(models.Model):
+    _name = 'hr.holidays.imposed'
+    _description = 'Manage imposed holidays'
+
+    name = fields.Char(required=True)
+    date = fields.Date(required=True)
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string="Company",
+        default=lambda self: self.env['res.company']._company_default_get(
+            'hr.holidays.imposed')
+    )
+    status_id = fields.Many2one(
+        comodel_name='hr.holidays.status',
+        string="Leave type",
+        required=True
+    )
+    employee_ids = fields.Many2many(comodel_name='hr.employee')
```

Comment:
can you add a `help` here mentionning that "if the field is left empty, then the holiday will apply to all employees of the company"?